From d279338ea0dab771a3c9c63642c334f2fb636457 Mon Sep 17 00:00:00 2001
From: Surej Joseph <surej.aj@gmail.com>
Date: Mon, 16 Mar 2020 10:54:34 -0500
Subject: [PATCH 3/4] FOS-QEMUX86_64: Increase the memory allocatable size in
 Uboot proper.

Inorder to decompress an LZO vmlinux binary we need to increase
the allocatable memory in Uboot. Increasing it to 32MB.
---
 include/configs/x86-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/x86-common.h b/include/configs/x86-common.h
index 4180b25f97..6d90f753e8 100644
--- a/include/configs/x86-common.h
+++ b/include/configs/x86-common.h
@@ -21,7 +21,7 @@
 
 #undef CONFIG_ZLIB
 #undef CONFIG_GZIP
-#define CONFIG_SYS_BOOTM_LEN		(16 << 20)
+#define CONFIG_SYS_BOOTM_LEN		(32 << 20)
 
 /* SATA AHCI storage */
 #ifdef CONFIG_SCSI_AHCI
-- 
2.17.1

